resource-watch/resource-watch
# Explore Near-real-time section

![image](https://user-images.githubusercontent.com/545342/76308645-822b9980-62cb-11ea-8e9e-a66179d8e9f6.png)

## Overview
This PR adds the Explore Near-real-time section

## Testing instructions
Go to http://localhost:9000/data/explore?section=Near%20Real-Time and verify that the right datasets load according to their type and date. (Also dataset cards should be fully functional)
